plugins/modules/packaging/language/cpanm.py

```diff
@@ -53,8 +54,11 @@
     type: str
   system_lib:
     description:
-     -  Use this if you want to install modules to the system perl include path. You must be root or have "passwordless" sudo for this to work.
-     -  This uses the cpanm commandline option '--sudo', which has nothing to do with ansible privilege escalation.
+      - Use this if you want to install modules to the system perl include path. You must be root or have "passwordless" sudo for this to work.
+      - This uses the cpanm commandline option C(--sudo), which has nothing to do with ansible privilege escalation.
+      - >
+        This option is not recommended for use and it should be deprecated in the future. If you need to escalate privileges
```

Comment:
"should", or "will"?
```suggestion
        This option is not recommended for use and it will be deprecated in the future. If you need to escalate privileges
```

Reply:
definitely will, thanks for spotting.